### DISCUSSION TITLE
设置明细表多行文本后如何调整行自适应文本显示？

### DISCUSSION THREAD

**Discussion by Awei-coder** [antvis/S2#2793] [Q&A]
调整前：
![image](https://github.com/antvis/S2/assets/65280616/c36bfc08-4f32-439f-b94d-9003c33eb5f0)
调整后：
![image](https://github.com/antvis/S2/assets/65280616/2d057cd2-8268-45c9-b33c-954a858c2b1c)
能否做到类似excel调整行动态显示文字？

**Comment by lijinke666** [comment]
目前不支持, 出于性能考虑, 目前是根据 `maxLines` 在初次渲染时来决定展示多少行, 你可以监听 resize 事件根据单元格高度动态调整 `maxLines` 的值.

后面考虑下该功能.

**Reply by Awei-coder** [reply]
好的，谢谢，这个功能还是挺常见的，希望后续能够支持('ᴗ' )

**Comment by lijinke666** [comment]
最新版已支持
